packages/@coorpacademy-components/src/atom/icon/index.js

```diff
@@ -63,7 +63,7 @@ const Icon = React.memo(function Icon({
       <FontAwesomeIcon
         icon={`fa-${iconName}`}
         color={effectiveIconColor}
-        font-size={effectiveSize.faSize}
+        fontSize={effectiveSize.faSize}
```

Comment:
(style comment: I personaly prefer kebab-case for attribute names in jsx, but that's a detail, and discutable)

Reply:
Ah! I fixed this to `fontSize` because unit test consider it an error and failed

![image](https://github.com/CoorpAcademy/components/assets/160047673/5d9a084d-352f-43af-b4a3-953d9b9d5397)

Reply:
👌